### ISSUE TITLE
Docs: 'Running the Tests' section is out of date 

### ISSUE THREAD

**Issue by stevewritescode** [saltstack/salt#51136]
### Description of Issue/Question
It looks like this docs section about setting up and running the Salt test suite is out-of-date: https://github.com/saltstack/salt/blob/develop/doc/topics/tutorials/writing_tests.rst

Quote from @gtmanfred from https://github.com/saltstack/salt/pull/51082/files#r246492565:

>The docs should really have instructions about using test-kitchen to run the test suite.
>
>There are some examples here https://kitchen.saltstack.com/docs/file/docs/gettingstarted.md
>
> but nothing specific to the salt test suite, which uses just `bundle exec kitchen verify` and can be configured with the runtests verifier to only run specific tests.
>
>https://kitchen.saltstack.com/docs/file/docs/runtests.md#tests

**Comment by Ch3LL** [comment]
I will need to bump this to next week

**Comment by Ch3LL** [comment]
as an update here. Since our tests have continued to change over the last 6 months we are going to wait until the migration to pytest is complete so we can ensure when we update the test documentation it is all accurate. I will be working with the test working group when that migration is complete to ensure we update all of the required documentation. Juts wanted to update the issue here.

**Comment by Ch3LL** [comment]
also to add to the information that we want to add:

- running tests with nox
- setting up kitchen-salt
- running tests with pytest

**Comment by Ch3LL** [comment]
Updated in #https://github.com/saltstack/salt/pull/55995

Cannot merge until https://github.com/saltstack/salt/pull/55745 is  merged as well. 

Also, I still need to update the kitchen-salt jenkins docs here: https://kitchen.saltstack.com/docs/file/docs/jenkins.md before we can close this.

**Comment by Ch3LL** [comment]
https://github.com/saltstack/kitchen-salt/pull/301

**Comment by Ch3LL** [comment]
@stevewritescode any chance you can give both those PRs a review?

**Comment by sagetherage** [comment]
@s0undt3ch is there another PR to address the Documentation?

**Comment by Ch3LL** [comment]
I believe this issue can be closed since we updated the Kitchen-Salt and Main Salt Documentation with the new nox and pytest approach for testing. Do you agree @s0undt3ch can we close this one out?

**Comment by s0undt3ch** [comment]
Yes, we can.
